Automattic/jetpack
# Remove Jumpstart

Removing everything jumpstart related.

~Basically informs me that I should first PR a change which just modifies minimum to hide Jumpstart from Jetpack, and then PR separately follow-ups to clean up the code.~ Got convinced by @dereksmart that this is solid. (p1560894750228800-slack-jetpack-dna). If further testing reveals surprises that take more than a day to fix, we can reassess and do the simple hiding instead.

Jumpstart might normally get disabled once some module toggles are changed. (Will still need to confirm if this is the case). If so, should we keep Jumpstart for those who didn't activate any features in the checklist, @jeffgolenski?

See inline comments for some todos.

Resolves https://github.com/Automattic/jetpack/issues/12601

#### Changes proposed in this Pull Request:
* Remove Jumpstart feature

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Removes existing feature.

#### Testing instructions:
- Create a new Jetpack site and connect it
- From Calypso checklist, click back to wp-admin without changing settings — you shouldn't see jumpstart and no errors, observe also PHP log.
- With another site, when in Calypso, turn on some features in settings and then return to wp-admin. Still no jumpstart and no errors. 
- Try different plan tiers

#### Proposed changelog entry for your changes:
- Remove feature to activate recommended features with one click
